components/notifier/src/notifier/notifier.py

```diff
@@ -56,12 +42,14 @@ def record_health() -> None:
         with open(filename, "w", encoding="utf-8") as health_check:
             health_check.write(datetime.now().isoformat())
     except OSError as reason:
-        logging.error("Could not write health check time stamp to %s: %s", filename, reason)
+        logging.error(
+            "Could not write health check time stamp to %s: %s", filename, reason)
```

Comment:
Please check your line width config, this should not have been broken across two lines.